src/client/pydaos/pydaos_core.py

```diff
@@ -22,6 +22,13 @@
     {key: value for key, value in list(pydaos_shim.__dict__.items())
      if key.startswith("OC_")})
 
+def _get_object_id(cid):
```

Comment:
flake wants 2 blank lines between top-level definitions
```suggestion

def _get_object_id(cid):
```